### PR TITLE
use google uuid; add module

### PR DESCRIPTION
**Pull request by brenol**

### CODE REVIEW

**Comment by garethlewin** [comment]
Thanks for this, need to update the travis file so that the right package is pulled. I'll make a separate pr to do a go module build once this is in and works.

**Comment by mikemoate** [comment]
@brenol @GarethLewin is there anything we can do to help get this PR in?

If not, I'd be happy to put up a new PR (based on this) with the travis file updated. I'd also be happy to help out with enabling the go module build.

We use this library a lot and would love to see it have go modules support.

**Comment by garethlewin** [comment]
Sorry, I have been very busy lately and this fell off my radar. Not a good reason and I'll make sure to find time soon, in the meantime such a PR would be very useful thanks @mikemoate or just a separate pr that fixes modules.

File: cmd/noopkinsumer/main.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/twinj/uuid"
+	"github.com/google/uuid"
```
Comment (garethlewin) [inline cmd/noopkinsumer/main.go:17]: Please update the travis file to pull this instead of the old one

**Comment by brenol** [comment]
Hey guys,

I can do those changes no problem. Was a little away for the time being but
I can fix the work done on this one.

I did not notice I had to change the travis file so sorry for that. I will
fix it later today :)

**Comment by taraspos** [comment]
@brenol any update here?
I was waiting for you, but I can create my own PR... This is a couple lines fix.

**Comment by brenol** [comment]
Ugh, sorry, please get to work on it as I há e been without time.
Sorry about that!

**Pull request closed without merging**